fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ConstantMaker.java

```diff
@@ -34,12 +34,12 @@ public ConstantMaker(MetricDictionary metricDictionary) {
     }
 
     @Override
-    public LogicalMetric make(String metricName, List<String> dependentMetrics) {
+    public LogicalMetric make(LogicalMetricInfo logicalMetricInfo, List<String> dependentMetrics) {
         // Check that we have the right number of metrics
-        assertRequiredDependentMetricCount(metricName, dependentMetrics);
+        assertRequiredDependentMetricCount(logicalMetricInfo.getName(), dependentMetrics);
 
         // Actually build the metric.
-        return makeInner(metricName, dependentMetrics);
+        return makeInner(logicalMetricInfo.getName(), dependentMetrics);
```

Comment:
Use the makeInner with logicalMetricInfo as argument instead of the string metric name version.